docs/index.md

```diff
@@ -198,3 +198,24 @@ Let’s take a look at few recommendations on how and where to run the chaos tes
 - Run multiple chaos tests at once to mimic the production outages
   - For example, hogging both IO and Network at the same time instead of running them separately to observe the impact.
   - You might have existing test cases, be it related to Performance, Scalability or QE, run the chaos in the background during the test runs to observe the impact. Signaling feature in Kraken can help with coordinating the chaos runs i.e start, stop, pause the scenarios based on the state of the other test jobs.
+
+
+#### Chaos testing within OpenShift Organization
```

Comment:
@paigerube14 Maybe change the title to Chaos testing in practice within OpenShift Organization? Thoughts?